.travis.yml

```diff
@@ -3,12 +3,11 @@ language: go
 sudo: false
 
 go:
-  - "1.9"
-  - "1.10"
-  - tip
```

Comment:
is tip the latest version on master? Why are we dropping this?

Reply:
Because it isn't very useful and it slows down CI to have to pull the latest master build every time.